src/plugins/imagery/components/ImageryViewLayout.vue

```diff
@@ -135,9 +135,14 @@
              :class="{ selected: focusedImageIndex === index && isPaused }"
              @click="setFocusedImage(index, thumbnailClick)"
         >
-            <img class="c-thumb__image"
-                 :src="image.url"
+            <a :href="image.url"
+               :download="image && image.filename || ''"
```

Comment:
I think this is the right general approach, but I don't think we should hard-code the attribute name. 

I think we need a new telemetry metadata hint here. Something like `imageDownloadName`. This could then be used to extract the correct file name from the telemetry datum, irrespective of how the value is named. We should also update our [Telemetry Metadata Value Hints API docs](https://github.com/nasa/openmct/blob/master/API.md#value-hints) to reflect this.